install.sh

```diff
@@ -0,0 +1,635 @@
+#!/bin/bash
+export LANG=en_US.UTF-8
+set -e
+
+LOGO=$(cat <<'EOF'
+    ___       ___       ___       ___       ___       ___       ___       ___       ___       ___   
+   /\  \     /\  \     /\  \     /\__\     /\  \     /\  \     /\  \     /\  \     /\  \     /\__\  
+  /::\  \   /::\  \   /::\  \   /:| _|_    \:\  \   /::\  \    \:\  \   /::\  \   /::\  \   /:/ _/_ 
+ /::\:\__\ /:/\:\__\ /::\:\__\ /::|/\__\   /::\__\ /\:\:\__\   /::\__\ /::\:\__\ /:/\:\__\ /::- \__\\
+ \/\::/  / \:\:\/__/ \:\:\/  / \/|::/  /  /:/\/__/ \:\:\/__/  /:/\/__/ \/\::/  / \:\ \/__/ \;:;- ,- 
+   /:/  /   \::/  /   \:\/  /    |:/  /   \/__/     \::/  /   \/__/      /:/  /   \:\__\    |:|  |  
+   \/__/     \/__/     \/__/     \/__/               \/__/               \/__/     \/__/     \|__|  
+EOF
+)
+
+APP_NAME="agentstack"
+VERSION="0.3.5"
+REPO_URL="https://github.com/AgentOps-AI/AgentStack"
+RELEASE_PATH_URL="$REPO_URL/archive/refs/tags"
+CHECKSUM_URL=""  # TODO
+PYTHON_VERSION=">=3.10,<3.13"
+UV_INSTALLER_URL="https://astral.sh/uv/install.sh"
+PYTHON_BIN_PATH=""  # set after a verified install is found
+DEV_BRANCH=""  # set by --dev-branch flag
+DO_UNINSTALL=0  # set by uninstall flag
+PRINT_VERBOSE=0
+PRINT_QUIET=1
+
+MSG_SUCCESS=$(cat <<EOF
+✅ Setup complete!
+
+To get started with AgentStack, run:
+    exec \$SHELL
+    agentstack init
+
+For more information, run:
+    agentstack docs -or- agentstack quickstart
+EOF
+)
+
+MSG_ALREADY_INSTALLED=$(cat <<EOF
+✅ AgentStack is already installed.
+Run 'agentstack update' to update to the latest version.
+EOF
+)
+
+MSG_UNINSTALL=$(cat <<EOF
+✅ AgentStack has been uninstalled.
+If you encountered any issues, or have feedback, please open an issue:
+    $REPO_URL/issues
+EOF
+)
+
+usage() {
+    cat <<EOF
+agentstack-install.sh
+
+The installer for AgentStack
+
+This script installs uv the Python package manager, installs a compatible Python 
+version ($PYTHON_VERSION), and installs AgentStack.
+
+USAGE:
+    agentstack-install.sh [OPTIONS]
+
+OPTIONS:
+    uninstall                  Uninstall
+    --version=<version>        Specify version to install (default: $VERSION)
+    --python-version=<version> Specify Python version to install (default: $PYTHON_VERSION)
+    --dev-branch=<branch>      Install from a specific git branch/commit/tag
+    --verbose                  Enable verbose output
+    --quiet                    Suppress output
+    -h, --help                 Show this help message
+EOF
+}
+# TODO allow user to specify install path with --target
+# TODO allow user to specify Python version with --python-version
+# TODO uninstall
```

Comment:
lets prioritize getting this finished. i think we can leave these TODOs in and add tickets for them to get this merged

Reply:
okay i see these are actually implemented below. if the todo is just to add an arg, send it

Reply:
just forgot to delete 'em 🤓